repo/maintenance/maintenance_params.go

```diff
@@ -23,6 +23,8 @@ type Params struct {
 	FullCycle  CycleParams `json:"full"`
 
 	LogRetention LogRetentionOptions `json:"logRetention"`
+
+	ExtendObjectLocks bool `json:"extendObjectLocks"`
```

Comment:
nit: explicitly set this to `false` in DefaultParams and add a comment for documentation purposes.

Reply:
I'm not sure what comment you are looking for?  The assignment seems to be self-evident to me.